Sources/DeepLearning/Layer.swift

```diff
@@ -1289,3 +1289,121 @@ public extension RNNCell {
         return applied(to: RNNCellInput(input: input, state: state))
     }
 }
+
+/// A Simple RNN Cell.
+public struct SimpleRNNCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
+    public var weight: Tensor<Scalar>
+    public var bias: Tensor<Scalar>
+
+    @noDerivative var stateShape: TensorShape
```

Comment:
I think this can be public. Same for LSTM.

Reply:
Sure thing.